public/video-ui/src/actions/VideoActions/videoPageCreate.js

```diff
@@ -24,12 +24,12 @@ function errorReceivingVideoPageCreate(error) {
   };
 }
 
-export function createVideoPage(id, title, composerUrl, data) {
+export function createVideoPage(id, metadata, composerUrl, data) {
```

Comment:
I've renamed the data to videoBlock because it is misleading. It gets posted to flex when video is added to the page and it gets parsed from the metadata before createVideoPage action is called